src/Test.UnitTests.Sarif/Core/ToolTests.cs

```diff
@@ -69,15 +69,12 @@ public void Tool_ParseFileVersion_ExtractsDottedQuadFileVersion()
 
                 if (!succeeded)
                 {
-                    sb.AppendLine($"    Input: {SafeFormat(testCase.Input)} Expected: {SafeFormat(testCase.ExpectedOutput)} Actual: {SafeFormat(actualOutput)}");
+                    sb.AppendLine($"    Input: {Utilities.SafeFormat(testCase.Input)} Expected: {Utilities.SafeFormat(testCase.ExpectedOutput)} Actual: {Utilities.SafeFormat(actualOutput)}");
                 }
             }
 
             sb.Length.Should().Be(0,
                 $"all test cases should pass, but the following test cases failed:\n{sb.ToString()}");
         }
-
-        private static string SafeFormat(string s)
```

Comment:
As I'm adding a null check, I have need of this.  It felt like it would be generally useful, so I moved it to the existing utilities class.  If there's a better place, or if you would prefer I just duplicate it, given it's size, just yell and I can undo\move.

Reply:
LGTM.

---
In reply to: [399076307](https://github.com/microsoft/sarif-sdk/pull/1831#discussion_r399076307) [](ancestors = 399076307)